README.md

```diff
@@ -3,161 +3,100 @@
 [![Build Status](https://travis-ci.org/weldr/welder-web.svg?branch=master)](https://travis-ci.org/weldr/welder-web)
 [![codecov](https://codecov.io/gh/weldr/welder-web/branch/master/graph/badge.svg)](https://codecov.io/gh/weldr/welder-web)
 
-The web interface for Composer!
+**The web interface for Composer!**
 
+Cockpit Composer is a common web interface and running as a [Cockpit](https://cockpit-project.org/) plugin. Cockpit Composer joins together with [Lorax Composer](https://github.com/weldr/lorax/tree/lorax-composer) to generate custom images suitable for deploying systems, or as images ready to upload the cloud.
 
-### Getting Started
+## Making changes on Cockpit Composer
 
-**Step 1**. Make sure that you have [Node.js](https://nodejs.org/) v6.9.1 or newer installed on your
-machine. For example:
-```shell
-nvm install 6.9.1
-```
-Or if it's installed, make sure you're using it:
-```shell
-nvm use 6.9.1
-```
+Here's where to get the code:
 
-**Step 2**. Clone this repository and install its dependencies:
+    $ git clone https://github.com/weldr/welder-web.git
+    $ cd welder-web/
 
-```shell
-$ git clone -o upstream -b master --single-branch https://github.com/weldr/welder-web
-$ cd welder-web/
-$ npm install                   # Install project dependencies listed in package.json
-```
+The remainder of the commands assume you're in the top level of the
+Cockpit Composer git repository checkout.
 
-**Step 3**. Compile and launch your app by running:
-
-```shell
-$ node run build
-$ node run                      # Same as `npm start` or `node run start`
-```
+### Getting the development dependencies
 
-You can also test your app in release (production) mode by running `node run start --release` or
-with HMR and React Hot Loader disabled by running `node run start --no-hmr`. The app should become
-available at [http://localhost:3000/](http://localhost:3000/).
+On Fedora or Red Hat Enterprise Linux:
 
-You can also enable code instrumentation with istanbul by specifying
-`node run build --with-coverage` or `node run --with-coverage`. This is needed if you want to
-collect code coverage from end-to-end tests. Don't use `--with-coverage` when running the unit tests.
-`jest` provides its own instrumentation and it will break if we do double instrumentation.
-Disabled by default!
+* First install Cockpit on your local machine as described in: https://cockpit-project.org/running.html.
+* Next install and start lorax-composer:
+```
+    $ sudo yum install lorax-compose
+    $ sudo systemctl start lorax-composer
+```
 
+Cockpit Composer uses Node.js during development. Node.js is not used at runtime. To make changes on Cockpit you'll want to install Node.js, NPM.
```

Comment:
This sentence would fit better in the next enumeration point, where it avoids the duplication.